pkgs/applications/editors/texstudio/default.nix

```
@@ -24,7 +24,8 @@ mkDerivation rec {
       LaTeX editing with completion, structure viewer, preview,
       spell checking and support of any compilation chain.
     '';
-    homepage = "http://texstudio.sourceforge.net";
+    homepage = "https://texstudio.org";
+    changelog = "https://github.com/texstudio-org/texstudio/blob/${version}/utilities/manual/CHANGELOG.txt";
     license = licenses.gpl2Plus;
     platforms = [ "x86_64-linux" ];
```

It seems this builds on macOS and my guess is it builds on aarch64-linux too  
```suggestion
    platforms = platforms.unix;
```

---

Does it actually run on darwin?

---

Hm, the derivation got built, but did not contain any binaries in $out

Maybe let's just use `platforms.linux` then